### DISCUSSION TITLE
Welcome to connect-four Discussions!

### DISCUSSION THREAD

**Discussion by nickytonline** [nickytonline/connect-four#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sadiejay** [comment]
Hello hello! I'm Sadie and I'm a maintainer here! Basically shadowing Nick and Jörn to get an idea of what it means to be a maintainer and also get my feet wet in svelte. 

I got some ideas and y'all will be seeing that shortly :)
